crawl-ref/source/dat/des/builder/uniques.des

```diff
@@ -305,6 +305,10 @@ NAME:   uniq_boris
 DEPTH:  Depths:3-, Vaults:2-, Crypt, Tomb
 : place_unique(_G, "Boris")
 
+NAME:   uniq_ciri
+DEPTH:  Abyss:2-, Depths:3-, Pan
```

Comment:
Abyss and Pan uniques generally aren't super threatening. It's pretty easy to run away from them, and then they're gone. Unless there's something that makes them extremely difficult to flee (even by using scrolls of teleportation) or something that makes the player really *want* to fight them, they're going to just be a 'noob trap' and not a lot else.

I'm also uncertain that it's going to work well to have the same enemies appear both in Depths and in Pan, since the latter is a significantly more dangerous place than the former. I'm aware that we already do this with Mennas, but I'm not particularly convinced that works well either. :)